src/modules/SystemSettings.rb

```diff
@@ -128,8 +130,13 @@ def Activate
         # set the scheduler
         Bootloader.modify_kernel_params("elevator" => new_elevator)
 
-        # TODO FIXME: set the scheduler for all disk devices,
-        # reboot is required to activate the new scheduler now
+        # activate the scheduler for all disk devices
+        if new_elevator != :missing
```

Comment:
BTW, as there were some issues to understand `why` and `how`, isn't this a good idea to document it directly in the code?